cloudpathlib/cloudpath.py

```diff
@@ -34,6 +37,14 @@
 from urllib.parse import urlparse
 from warnings import warn
 
+from cloudpathlib.typing import (
+    OpenBinaryMode,
+    OpenBinaryModeReading,
+    OpenBinaryModeUpdating,
+    OpenBinaryModeWriting,
+    OpenTextMode,
+)
+
```

Comment:
Import these directly from `_typeshed` instead of vendoring:

```python
if TYPE_CHECKING:
    from _typeshed import OpenBinaryMode, OpenBinaryModeReading, OpenBinaryModeUpdating, OpenBinaryModeWriting, OpenTextMode
```

You'll need to put the references in quotations since they won't be available at runtime.

Reply:
Done!